### ISSUE TITLE
Ambient or HemisphereLight not working in deferred renderer

### ISSUE THREAD

**Issue by martinglish** [mrdoob/three.js#6450]
The deferred renderer is a star, great job! However, I can't get any kind of ambient light to work. I checked the source code and can't find any reference to creating a proxy for an ambient light but did see a reference to a proxy for a HemisphereLight . When I add one my scene goes from this (WebGLRenderer)...

![r1](https://cloud.githubusercontent.com/assets/1647664/7316379/6a86cbea-ea6e-11e4-8307-db02d9f27f49.PNG)

to this (WebGLDeferredRenderer)...

![r2](https://cloud.githubusercontent.com/assets/1647664/7316393/807bf4ac-ea6e-11e4-8ddd-057be1ae8310.PNG)

There are no other lights in the scene, all the geometry has the lighting baked onto the texture maps.
Other lights are working OK.

Thanks.

**Comment by mrdoob** [comment]
If all the lights are backed in textures, why do you need WebGLDeferredRenderer?

**Comment by martinglish** [comment]
Ah, well. I'm experimenting with making the lighting dynamic. The 
application is used to visualise artwork in a gallery which means making 
the scene generally darker and having a spotlight focus on each piece.

**Comment by martinglish** [comment]
Quick update, I tried adding a HemiSphere light to one of the standard deferred rendering demos and it did the same sort of thing.

**Comment by martinglish** [comment]
Actually this turned out to be more serious than I thought at first. The more I looked at the scene in the deferred renderer the more I realised what had happened was that all the texture maps had gone and the geometry was being rendered in "grey" apart from where the material had an emissive setting. The fix was to move from r70 to r71. 

Should the deferred renderer support transparent materials does anybody know?

Thanks
